### PR TITLE
Merge 4.7.4

### PR DESCRIPTION
**Pull request by nonemax**

### CODE REVIEW

File: build/pgo-deployer/Dockerfile [hunk 1]
```diff
@@ -50,6 +50,8 @@ RUN set -ex; \
 
 RUN mkdir -p /opt/cpm/bin
 
+RUN mkdir -p /opt/cpm/bin
```
Comment (hors) [inline build/pgo-deployer/Dockerfile:53]: why do we need it (two the same lines)?

**Review by hors — CHANGES_REQUESTED** [review]

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | failed |
| demand-backup | passed |
| scaling | passed |
| recreate | passed |
| affinity | passed |
| scheduled-backup | passed |
| monitoring | passed |
| self-healing | passed |
| operator-self-healing | passed |
| clone-cluster | passed |
| tls-check | passed |
| upgrade | passed |
| smart-update | passed |
| version-service | passed |
| users | passed |
| ns-mode | passed |

commit: https://github.com/percona/percona-postgresql-operator/pull/186/commits/d2067126d1b5e5aa2dc033daf7a5e2404e82cd50
image: `perconalab/percona-postgresql-operator:PR-186-d2067126-pgo-apiserver`

image: `perconalab/percona-postgresql-operator:PR-186-d2067126-pgo-event`

image: `perconalab/percona-postgresql-operator:PR-186-d2067126-pgo-rmdata`

image: `perconalab/percona-postgresql-operator:PR-186-d2067126-pgo-scheduler`

image: `perconalab/percona-postgresql-operator:PR-186-d2067126-postgres-operator`

image: `perconalab/percona-postgresql-operator:PR-186-d2067126-pgo-deployer`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | failed |
| demand-backup | passed |
| scaling | passed |
| recreate | passed |
| affinity | passed |
| scheduled-backup | passed |
| monitoring | passed |
| self-healing | passed |
| operator-self-healing | passed |
| clone-cluster | passed |
| tls-check | passed |
| upgrade | passed |
| smart-update | passed |
| version-service | passed |
| users | passed |
| ns-mode | passed |

commit: https://github.com/percona/percona-postgresql-operator/pull/186/commits/bf4749c31a99e931ef47daeea4f384d0cb23cc71
image: `perconalab/percona-postgresql-operator:PR-186-bf4749c3-pgo-apiserver`

image: `perconalab/percona-postgresql-operator:PR-186-bf4749c3-pgo-event`

image: `perconalab/percona-postgresql-operator:PR-186-bf4749c3-pgo-rmdata`

image: `perconalab/percona-postgresql-operator:PR-186-bf4749c3-pgo-scheduler`

image: `perconalab/percona-postgresql-operator:PR-186-bf4749c3-postgres-operator`

image: `perconalab/percona-postgresql-operator:PR-186-bf4749c3-pgo-deployer`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| demand-backup | passed |
| init-deploy | failed |
| scaling | passed |
| recreate | passed |
| affinity | passed |
| scheduled-backup | passed |
| monitoring | passed |
| self-healing | passed |
| operator-self-healing | passed |
| clone-cluster | passed |
| tls-check | passed |
| upgrade | passed |
| smart-update | passed |
| version-service | passed |
| users | passed |
| ns-mode | passed |

commit: https://github.com/percona/percona-postgresql-operator/pull/186/commits/c71436c5f8bcee41ecbc6d51fbf59a502bfe7352
image: `perconalab/percona-postgresql-operator:PR-186-c71436c5-pgo-apiserver`

image: `perconalab/percona-postgresql-operator:PR-186-c71436c5-pgo-event`

image: `perconalab/percona-postgresql-operator:PR-186-c71436c5-pgo-rmdata`

image: `perconalab/percona-postgresql-operator:PR-186-c71436c5-pgo-scheduler`

image: `perconalab/percona-postgresql-operator:PR-186-c71436c5-postgres-operator`

image: `perconalab/percona-postgresql-operator:PR-186-c71436c5-pgo-deployer`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | failed |
| demand-backup | failed |
| scaling | failed |
| scheduled-backup | failed |
| recreate | failed |
| affinity | failed |
| monitoring | failed |
| self-healing | failed |
| operator-self-healing | failed |
| clone-cluster | failed |
| tls-check | failed |
| upgrade | failed |
| smart-update | failed |
| version-service | failed |
| users | failed |
| ns-mode | failed |

commit: https://github.com/percona/percona-postgresql-operator/pull/186/commits/c97c68cd35c1b89431093f77624f2e4d51ffae0e
image: `perconalab/percona-postgresql-operator:PR-186-c97c68cd-pgo-apiserver`

image: `perconalab/percona-postgresql-operator:PR-186-c97c68cd-pgo-event`

image: `perconalab/percona-postgresql-operator:PR-186-c97c68cd-pgo-rmdata`

image: `perconalab/percona-postgresql-operator:PR-186-c97c68cd-pgo-scheduler`

image: `perconalab/percona-postgresql-operator:PR-186-c97c68cd-postgres-operator`

image: `perconalab/percona-postgresql-operator:PR-186-c97c68cd-pgo-deployer`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| demand-backup | failed |
| scheduled-backup | failed |

commit: https://github.com/percona/percona-postgresql-operator/pull/186/commits/c97c68cd35c1b89431093f77624f2e4d51ffae0e
image: `perconalab/percona-postgresql-operator:PR-186-c97c68cd-pgo-apiserver`

image: `perconalab/percona-postgresql-operator:PR-186-c97c68cd-pgo-event`

image: `perconalab/percona-postgresql-operator:PR-186-c97c68cd-pgo-rmdata`

image: `perconalab/percona-postgresql-operator:PR-186-c97c68cd-pgo-scheduler`

image: `perconalab/percona-postgresql-operator:PR-186-c97c68cd-postgres-operator`

image: `perconalab/percona-postgresql-operator:PR-186-c97c68cd-pgo-deployer`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| demand-backup | failed |
| scheduled-backup | failed |

commit: https://github.com/percona/percona-postgresql-operator/pull/186/commits/c97c68cd35c1b89431093f77624f2e4d51ffae0e
image: `perconalab/percona-postgresql-operator:PR-186-c97c68cd-pgo-apiserver`

image: `perconalab/percona-postgresql-operator:PR-186-c97c68cd-pgo-event`

image: `perconalab/percona-postgresql-operator:PR-186-c97c68cd-pgo-rmdata`

image: `perconalab/percona-postgresql-operator:PR-186-c97c68cd-pgo-scheduler`

image: `perconalab/percona-postgresql-operator:PR-186-c97c68cd-postgres-operator`

image: `perconalab/percona-postgresql-operator:PR-186-c97c68cd-pgo-deployer`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| demand-backup | failed |
| init-deploy | failed |
| scheduled-backup | failed |
| scaling | failed |
| recreate | failed |
| affinity | failed |
| monitoring | failed |
| self-healing | failed |
| operator-self-healing | failed |
| clone-cluster | failed |
| tls-check | failed |
| upgrade | failed |
| smart-update | failed |
| version-service | failed |
| users | failed |
| ns-mode | failed |

commit: https://github.com/percona/percona-postgresql-operator/pull/186/commits/c97c68cd35c1b89431093f77624f2e4d51ffae0e
image: `perconalab/percona-postgresql-operator:PR-186-c97c68cd-pgo-apiserver`

image: `perconalab/percona-postgresql-operator:PR-186-c97c68cd-pgo-event`

image: `perconalab/percona-postgresql-operator:PR-186-c97c68cd-pgo-rmdata`

image: `perconalab/percona-postgresql-operator:PR-186-c97c68cd-pgo-scheduler`

image: `perconalab/percona-postgresql-operator:PR-186-c97c68cd-postgres-operator`

image: `perconalab/percona-postgresql-operator:PR-186-c97c68cd-pgo-deployer`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| demand-backup | failed |
| init-deploy | failed |
| scheduled-backup | failed |
| scaling | failed |
| recreate | failed |
| affinity | failed |
| monitoring | failed |
| self-healing | failed |
| operator-self-healing | failed |
| clone-cluster | failed |
| tls-check | failed |
| upgrade | failed |
| smart-update | failed |
| version-service | failed |
| users | failed |
| ns-mode | failed |

commit: https://github.com/percona/percona-postgresql-operator/pull/186/commits/c97c68cd35c1b89431093f77624f2e4d51ffae0e
image: `perconalab/percona-postgresql-operator:PR-186-c97c68cd-pgo-apiserver`

image: `perconalab/percona-postgresql-operator:PR-186-c97c68cd-pgo-event`

image: `perconalab/percona-postgresql-operator:PR-186-c97c68cd-pgo-rmdata`

image: `perconalab/percona-postgresql-operator:PR-186-c97c68cd-pgo-scheduler`

image: `perconalab/percona-postgresql-operator:PR-186-c97c68cd-postgres-operator`

image: `perconalab/percona-postgresql-operator:PR-186-c97c68cd-pgo-deployer`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| demand-backup | passed |
| scheduled-backup | passed |

commit: https://github.com/percona/percona-postgresql-operator/pull/186/commits/c97c68cd35c1b89431093f77624f2e4d51ffae0e
image: `perconalab/percona-postgresql-operator:PR-186-c97c68cd-pgo-apiserver`

image: `perconalab/percona-postgresql-operator:PR-186-c97c68cd-pgo-event`

image: `perconalab/percona-postgresql-operator:PR-186-c97c68cd-pgo-rmdata`

image: `perconalab/percona-postgresql-operator:PR-186-c97c68cd-pgo-scheduler`

image: `perconalab/percona-postgresql-operator:PR-186-c97c68cd-postgres-operator`

image: `perconalab/percona-postgresql-operator:PR-186-c97c68cd-pgo-deployer`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| demand-backup | passed |
| scheduled-backup | passed |
| scaling | passed |
| recreate | passed |
| affinity | passed |
| monitoring | passed |
| self-healing | passed |
| operator-self-healing | passed |
| clone-cluster | passed |
| tls-check | passed |
| upgrade | passed |
| smart-update | passed |
| version-service | passed |
| users | passed |
| ns-mode | passed |

commit: https://github.com/percona/percona-postgresql-operator/pull/186/commits/c97c68cd35c1b89431093f77624f2e4d51ffae0e
image: `perconalab/percona-postgresql-operator:PR-186-c97c68cd-pgo-apiserver`

image: `perconalab/percona-postgresql-operator:PR-186-c97c68cd-pgo-event`

image: `perconalab/percona-postgresql-operator:PR-186-c97c68cd-pgo-rmdata`

image: `perconalab/percona-postgresql-operator:PR-186-c97c68cd-pgo-scheduler`

image: `perconalab/percona-postgresql-operator:PR-186-c97c68cd-postgres-operator`

image: `perconalab/percona-postgresql-operator:PR-186-c97c68cd-pgo-deployer`

**Review by hors — APPROVED** [review]

**Review by cap1984 — APPROVED** [review]

**Pull request merged**